### PR TITLE
Allow all Elements panels to be scrollable

### PR DESCRIPTION
**Pull request by abstractalgo**
The change affects:

- Event Listeners panel
- Layout panel

Before:

https://user-images.githubusercontent.com/1355455/150826526-a8bc97e7-0ef7-4b4a-ba43-76d399707afa.mp4

After:

https://user-images.githubusercontent.com/1355455/150826180-2a9ec885-30b1-46e6-bb4d-5c5149d90e69.mp4

### CODE REVIEW

**Comment by abstractalgo** [comment]
Tagged @jaril because of the Accordion stuff (might not be super relevant).

**Review by hbenl — APPROVED** [review]

File: src/devtools/client/debugger/src/components/shared/Accordion.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
   display: flex;
   flex-direction: column;
   overflow-y: hidden;
-  height: 100%;
+  min-height: 100%;
```
Comment (jcmorrow) [inline src/devtools/client/debugger/src/components/shared/Accordion.css:13]: This does break (or at least significantly change) the behavior of the pause panel accordion.

Comment (jcmorrow) [reply src/devtools/client/debugger/src/components/shared/Accordion.css:13]: Production is on the right, the preview for this branch is on the left.

https://user-images.githubusercontent.com/5903784/150836580-7b24f966-6eae-4a25-83cb-b17f3e480288.mp4

Comment (jcmorrow) [reply src/devtools/client/debugger/src/components/shared/Accordion.css:13]: I think that the preview branch version is a bit easier to use, but also feels broken if you are expecting a true accordion. We can always add a special case to either of these to keep them working.

Comment (abstractalgo) [reply src/devtools/client/debugger/src/components/shared/Accordion.css:13]: Great catch, thanks! 🙌 

I'll reconsider what might be the best approach (specific selector for Layout accordion only; or something else).

**Comment by abstractalgo** [comment]
Closing in favor of #5128 .

**Pull request closed without merging**